docs/index.md

```diff
@@ -178,6 +178,21 @@ plugins:
           ...
 ```
 
+You can also define the category-specific settings in a YAML file. To do that, you need to add a new default key `config_filename` in the plugin configuration:
```

Comment:
It might be better to explain the necessity to have a separate configuration file here - maybe "if you have lots of categories, you might want to put their configuration in a separate file to keep `mkdocs.yml` clean"?